### PR TITLE
Fixes format of entry value if needed

### PR DESCRIPTION
**Pull request by avonville**
Updated fix for[ issue-61](https://github.com/BoldGrid/weforms-pro/issues/61) in pro.

To test:

Create a form with a radio, multiselect, or checkbox field.
Submit the form and check the entry.
Below is the expected output

image

### CODE REVIEW

File: includes/class-notification.php [hunk 1]
```diff
@@ -463,33 +463,48 @@ public function get_merge_value( $tag ) {
     }
 
     /**
-     * Parse out the custom fields with entry meta values
+     * Parse out the custom fields with options or values. Since the options are what is stored, options may need to be
+     * used to find the value from the field settings.
```
Comment (bwmarkle) [inline includes/class-notification.php:467]: Can we include a specific example?

```php
* For example, let's say we have the following options:
* DEPARTMENT / EMAIL
* Support / support@example.com
* Sales / sales@example.com
* 
* Users need the ability to pass {field:department} and get "Support",
* and {value:department} to get support@email.com
```

File: includes/class-form-entry-manager.php [hunk 2]
```diff
@@ -51,4 +51,54 @@ public function all() {
     public function get( $entry_id ) {
         return new WeForms_Form_Entry( $entry_id, $this->form );
     }
+
+    /**
+     * Format Entry Value.
+     *
+     * @static
```
Comment (jamesros161) [inline includes/class-form-entry-manager.php:58]: Just a comment, you should usually put the @static tag last. It's not a requirement, but it is customary. From what I've seen anyways.

File: includes/class-form-entry-manager.php [hunk 3]
```diff
@@ -51,4 +51,54 @@ public function all() {
     public function get( $entry_id ) {
         return new WeForms_Form_Entry( $entry_id, $this->form );
     }
+
+    /**
+     * Format Entry Value.
+     *
+     * @static
+     * @param  array $field
```
Comment (jamesros161) [inline includes/class-form-entry-manager.php:59]: Your @param is missing the description. Format is 

@param [type] [variable_name] [description].

File: includes/class-form-entry-manager.php [hunk 4]
```diff
@@ -51,4 +51,54 @@ public function all() {
...
+     * @static
+     * @param  array $field
+     *
+     * @return array $field
+     */
+    public static function format_entry_value( $field ) {
+        switch ( $field['template'] ) {
+
+            case 'radio_field':
+                $value = array_search( $field['value'], $field['options'] );
```
Comment (jamesros161) [inline includes/class-form-entry-manager.php:67]: '=' not alligned

File: includes/class-form-entry.php [hunk 5]
```diff
@@ -421,4 +421,10 @@ public function get_payment_data() {
 
         return $wpdb->get_row( "SELECT * FROM {$wpdb->prefix}weforms_payments WHERE entry_id = {$this->id} " );
     }
+
+    public static function get_form_id( $entry_id ) {
```
Comment (jamesros161) [inline includes/class-form-entry.php:425]: MIssing function docblock. When you do add the docblock, be sure to add the @global tag. 
see https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-functions-class-methods

File: includes/class-notification.php [hunk 6]
```diff
@@ -463,30 +463,53 @@ public function get_merge_value( $tag ) {
...
+     * For example, let's say we have the following options:
+     * DEPARTMENT / EMAIL
+     * Support / support@example.com
+     * Sales / sales@example.com
+     *
+     * Users need the ability to pass {field:department} and get "Support",
+     * and {value:department} to get support@email.com
      *
      * @param string $text
+     * @param int    $entry_id
```
Comment (jamesros161) [inline includes/class-notification.php:478]: missing descriptions in doc block @param , and @return description

**Review by jamesros161 — CHANGES_REQUESTED** [review]

**Review by jamesros161 — APPROVED** [review]

**Pull request merged**